_includes/snippets_library/learnhpc_slurm/modules/wrong-gcc-version.snip

```diff
@@ -0,0 +1,79 @@
+<!---
```

Comment:
If this snippet is meant to render nothing, please check in a much smaller file.

Reply:
So the thing is that the Magic Castle tool can (currently) have two software stacks, the Compute Canada one and the EESSI one. For EESSI (which I'm working on), this snippet is not valid because we don't use a module hierarchy but the whole section is still valid for Magic Castle with the Compute Canada stack.

Having said that, I could say all that in the comment instead, so I will update this.